content/paths.yml

```
@@ -225,6 +225,16 @@
   delete:
     "$ref": "./paths/folder_watermarks__delete_folders_id_watermark.yml"
 
+"/folder_locks":
+  get:
+    "$ref": "./paths/folder_lock__get_folder_locks.yml"
+  post:
+    "$ref": "./paths/folder_lock__post_folder_locks.yml"
```

```suggestion
    "$ref": "./paths/folder_locks__get_folder_locks.yml"
  post:
    "$ref": "./paths/folder_locks__post_folder_locks.yml"
```
Make it match the tag